lms/djangoapps/instructor/views/api.py

```diff
@@ -1196,6 +1218,13 @@ def generate_registration_codes(request, course_id):
         internal_reference=internal_reference,
         customer_reference_number=customer_reference_number
     )
+    unit_price = (Decimal(sale_price)/course_code_number).quantize(Decimal('.01'), rounding=ROUND_DOWN)
+    invoice_item = CourseRegistrationCodeInvoiceItem.objects.create(
+        invoice=sale_invoice,
+        qty=course_code_number,
+        unit_price=unit_price,
```

Comment:
As I'm thinking about this more, I'm not sure I want `InvoiceItem` to know how to determine the "actual" price, since this can vary based on the item being sold.

For this reason, I think it makes sense to:
1) Continue using `unit_price` as the price-per-unit sold (same as you are already doing).
2) Remove the `billed_unit_price` field.